packages/block-library/src/query-loop/utils.js

```diff
@@ -0,0 +1,20 @@
+/* global wp */
+/**
+ * Returns arguments inherited from the template-type.
+ */
+export const getTemplateQuery = () => {
+	const templateType = wp.data.select( 'core/edit-site' ).getTemplateType();
+	if ( 'wp_template' === templateType ) {
+		const templateId = wp.data.select( 'core/edit-site' ).getTemplateId();
```

Comment:
`wp.data` shouldn't be referenced in Gutenberg this way. Using globals is reserved for plugins only for the simplicity of usage. `select` call can get outdated, without `useSelect` the component won't rerender when the value changes.